sdk/resourcemanager/azure-resourcemanager-appservice/src/main/java/com/azure/resourcemanager/appservice/implementation/FunctionAppsImpl.java

```diff
@@ -124,4 +134,48 @@ public FunctionAppImpl define(String name) {
     public Mono<Void> deleteByResourceGroupAsync(String groupName, String name) {
         return this.inner().deleteAsync(groupName, name);
     }
+
+    @Override
+    public Flux<String> deleteByIdsAsync(Collection<String> ids) {
```

Comment:
For delete operations, do the users need Flux<String> as return type?

Reply:
I think this is a list of IDs that got deleted. As method with `DelayError`, error will be after Flux of IDs.

Reply:
Got it. So users would get IDs returned, which are success deletion, and errors if applicable returned after all completed.